eng/pipelines/apiview-review-gen-python.yml

```diff
@@ -0,0 +1,44 @@
+pr: none
+
+trigger: none
+
+pool:
+  name: azsdk-pool-mms-ubuntu-2004-general
+  vmImage: MMSUbuntu20.04
+
+parameters:
+  - name: Reviews
```

Comment:
Later in your script you expect this to be a json string what is the expected data to be included in this?

Reply:
Might be worth giving a default value as example of the expected structure of this string.

Reply:
done